_security/access-control/users-roles.md

```diff
@@ -218,3 +218,40 @@ Map the role to your user:
 1. Choose the **Mapped users** tab and **Manage mapping**.
 1. For **Internal users**, add your bulk access user.
 1. Choose **Map**.
+
+## SuperAdmin role
+
+The `superAdmin` role enables adding, updating, and deleting reserved configurations like roles, roles_mapping, internal_users, action_groups, and tenants. Previously, these configurations were loaded from default YAML files. Now, the `superAdmin` role now has the privilege to add, update, and delete by using API calls.
+
+### Example API calls:
+Following are examples of API calls for `superAdmin` actions:
+
+Adding a Reserved Configuration:
+```
+curl -X PUT https://localhost:9200/_opendistro/_security/api/roles/new_role -k -H 'Content-Type: application/json' -d '{
```

Comment:
also don't user need to pass Admin certificate for invoking this API, other than invoking locally from the node? 
without identity to this API, system won't know the caller is super admin. And identity for super admin was being passed as client certificate with dn name which is allowlisted in config.yml 

Can you verify working these?